src/test/munit/socket-custom-protocol-config.xml

```diff
@@ -46,8 +46,8 @@
         </munit:validation>-->
     </munit:test>
 
-    <flow name="listenerFlow">
-        <sockets:listener config-ref="SocketsListenerConfigCustomTcp"/>
-        <munit-tools:queue />
-    </flow>
+<!--    <flow name="listenerFlow">-->
```

Comment:
can this comment be deleted?

Reply:
this is related to SOCON-9 so the whole test is disabled until it's fixed, I think